File: Assembler/LB 2/Task2.js
// Даны три числа. Найти произведение наименьшего и наибольшего числа. 

const a = 10
const b = 20
const c = 30

let min, max

if (a >= b && a >= c) {
    max = a
} else if (b >= a && b >= c) {
    max = b
} else if (c >= a && c >= b) {
    max = c
}

if (a <= b && a <= c) {
    min = a
} else if (b <= a && b <= c) {
    min = b
} else if (c <= a && c <= b) {
    min = c
}

const mult = max * min

console.log(mult)

File: Programming Methods/lab4/Task2/Task2.java
package Task2;

import java.lang.Math;
import java.util.stream.*;

public class Task2 {
    public static void main(String[] args) {
        int result = IntStream.range(10, 100)
        .filter(w -> {
            int a = w % 10;
            int b = w / 10;
            int max = Math.max(a, b);
            int min = Math.min(a, b);
            return (w == (a + b) * (max - min));
        })
        .findFirst()
        .orElse(-1);

        System.out.printf("This number is %s\n", result);
    }
}
